### DISCUSSION TITLE
Formatting rules from a .scss-lint.yml file?

### DISCUSSION THREAD

**Discussion by cssagogo** [sibiraj-s/vscode-scss-formatter#6] [Feature request]
The default formatting is conflicting with my .scss-lint.yml rules.  Is there a way to configure the rules of SCSS Formatter?

**Comment by sibiraj-s** [comment]
The extension uses prettier under the hood and Prettier is heavily opinionated. Not may rules are supported by prettier in-terms of SASS/CSS formatting, but there are some. I have exposed them https://github.com/sibiraj-s/vscode-scss-formatter#extension-settings.

Let me know if something is missing that prettier supports. I can add.

**Reply by sibiraj-s** [reply]
@cssagogo still facing the issue?
